scripts/setup.sh

```diff
@@ -200,9 +200,10 @@ elif [ "$VAGRANT" = true ]; then
   export XVFB_PREFIX="/usr/bin/xvfb-run"
   # Enforce proper ownership on oppia, oppia_tools, and node_modules or else NPM installs will fail.
   sudo chown -R vagrant.vagrant /home/vagrant/oppia /home/vagrant/oppia_tools /home/vagrant/node_modules
-elif [ -f "/usr/bin/google-chrome" ]; then
+
+elif [ -f "/usr/bin/chromium-browser" ]; then
   # Unix.
-  export CHROME_BIN="/usr/bin/google-chrome"
+  export CHROME_BIN="/usr/bin/chromium-browser"
```

Comment:
Please don't eliminate the chrome clause altogether. This will break things for users who don't have chromium but who do have chrome.

I think it is fine to have both clauses.